package.json

```diff
@@ -3,13 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.7.1",
+    "@emotion/styled": "^11.6.0",
```

Comment:
I added this dependencies because of compile error

_Failed to compile.
./node_modules/@mui/styled-engine/index.js
Module not found: Can't resolve '@emotion/react' in '/Users/alexnafangermanhotmail.com/web/internship-boilerplate/node_modules/@mui/styled-engine'_